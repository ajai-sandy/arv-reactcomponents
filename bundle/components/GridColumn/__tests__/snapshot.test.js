import React from "react";
import { mount, shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import GridColumnTest from "../test.component";

describe("Root Component Tests", () => {
  it("Matches Shallow SnapShot", () => {
    expect(shallowToJson(shallow(<GridColumnTest />))).toMatchSnapshot();
  });

  it("Matches Mounted SnapShot", done => {
    const mounted = mount(<GridColumnTest />);

    setTimeout(() => {
      expect(mounted).toMatchSnapshot();
      done();
    }, 300);
  });
});
