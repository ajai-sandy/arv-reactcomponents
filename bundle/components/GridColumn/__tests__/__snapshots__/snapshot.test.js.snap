// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Tests Matches Mounted SnapShot 1`] = `
<GridColumnTest>
  <GridColumn
    className=""
  >
    <div
      className="nwc-grid-col "
    >
      <GridColumn
        className=""
      >
        <div
          className="nwc-grid-col "
        >
          Hello
        </div>
      </GridColumn>
    </div>
  </GridColumn>
</GridColumnTest>
`;

exports[`Root Component Tests Matches Shallow SnapShot 1`] = `
<GridColumn
  className=""
>
  <GridColumn
    className=""
  >
    Hello
  </GridColumn>
</GridColumn>
`;
